### PR TITLE
feat(compiler): json static methods

### PR DESCRIPTION
**Pull request by hasanaburayyan**
This PR introduces Json static methods. It required some creativity with the grammar to use the `Json` keyword for both static methods and instantiating Json objects.

Closes: https://github.com/winglang/wing/issues/1679

*note*: This PR does not include `from_json()` methods on container types. Need a better way to ensure type safety with separate issue: https://github.com/winglang/wing/issues/1796

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: libs/wingsdk/src/std/json.ts [hunk 1]
```diff
@@ -2,6 +2,98 @@
...
+
+  /**
+   * Formats Json as string
+   *
+   * @macro (JSON.stringify($args$))
+   *
+   * @param json to format as string
+   * @returns string representation of the Json
+   */
+  public static toStr(json: Json): string {
```
Comment (eladb) [inline libs/wingsdk/src/std/json.ts:79]: I think the spec has some options here like `indent`

Comment (hasanaburayyan) [reply libs/wingsdk/src/std/json.ts:79]: Ah shoot yea I missed that example, I’ll update. I’ll go though and double check the examples as well to make sure the rest match.

File: libs/wingsdk/src/std/json.ts [hunk 2]
```diff
@@ -2,6 +2,98 @@
...
+
+  /**
+   * Parse a string into a Json
+   *
+   * @macro (JSON.parse($args$))
+   *
+   * @param str to parse as Json
+   * @returns Json representation of the string
+   */
+  public static parse(str: string): Json {
```
Comment (staycoolcall911) [inline libs/wingsdk/src/std/json.ts:92]: Still need to implement `.from_json()` methods for our primitive and container types.

**Comment by eladb** [comment]
@hasanaburayyan when this is ready for another review, please make sure to hit these to request a re-review:

Screenshot 2023-03-13 at 22 32 15

**Comment by hasanaburayyan** [comment]
> @hasanaburayyan when this is ready for another review, please make sure to hit these to request a re-review:
> 
> Screenshot 2023-03-13 at 22 32 15

👍 I wanted to get a tracking issue for some static methods on container types and kept getting sidetracked 😅

**Review by MarkMcCulloh — APPROVED** [review]

**Pull request merged**